docs/source/index.md

```diff
@@ -1,21 +1,99 @@
-# Welcome to MALA's documentation!
+![image](./img/logos/mala_horizontal.png)
+
+# Welcome to MALA!
+
+## What is MALA?
+
+MALA is a software package for building ML models that replace
+density functional theory (DFT) calculations. DFT is **the** most
+popular method for simulating materials on a quantum level and predicting 
+materials properties, and employed by researcher throughout the world. 
+
+While DFT is very efficient compared to other quantum mechanical simulation
+techniques, it still scales cubically with number of atoms and temperature 
+simulated, making large scale simulation at arbitrary temperatures very 
+computationally challenging. 
+
+Therefore, a new direction of research tries to [combine the power of ML with DFT](https://journals.aps.org/prmaterials/abstract/10.1103/PhysRevMaterials.6.040301).
+MALA is at the forefront of these efforts, being the first library to allow 
+users to build ML-DFT models that give **full access to the electronic structure**,
+observables such as the total free energy of the system **and** scale well with
+system size and temperature.
+
```

Comment:
Recent research activities focus on harnessing the [potential of ML and DFT](https://journals.aps.org/prmaterials/abstract/10.1103/PhysRevMaterials.6.040301). MALA leads these efforts by facilitating the construction of ML-DFT models, offering comprehensive access to the electronic structure and observables, including the total free energy of the system. Moreover, MALA ensures scalability with system size and various parameters, such as temperature.